corehq/apps/domain/forms.py

```diff
@@ -466,6 +467,8 @@ def clean_default_timezone(self):
 
     def save(self, request, domain):
         try:
+            domain.hr_name = self.cleaned_data['hr_name']
+
```

Comment:
Yeah, I think so.  Here are some thoughts on exception handling:
* Generally you should only catch exceptions that you expect (so never `Exception`), that way you don't accidentally hide real problems.
* For this same reason, the `try` block should be as small as possible, so if the concern is that writing that image will fail, then only that part should be in a try block, and the exception to catch is `IOErrror` or whatever happens when writing fails. 
* In python, you can do try/except/else to only execute the rest of the code if there wasn't an exception:
```python
try:
    thing = call_that_might_fail()
except ThingException as e:
    handle_error(e)
else:
    # cool, it worked!
    do_other_stuff(thing)
```
(I'm not sure how familiar you are with python, so forgive me if this is old news)